src/components/PathwaysList/PathwaysTable.tsx

```diff
@@ -14,69 +14,92 @@ import {
 
 import { usePathwayContext } from 'components/PathwayProvider';
 import { downloadPathway } from 'utils/builder';
+import PathwayModal from './PathwayModal';
 
 import useStyles from './styles';
+import { Pathway } from 'pathways-model';
 
 const PathwaysTable: FC = () => {
   const styles = useStyles();
   const { pathways, deletePathway } = usePathwayContext();
+  const [open, setOpen] = useState(false);
+  const [editablePathway, setEditablePathway] = useState<Pathway>();
+
+  function openEditPathwayModal(pathway: Pathway): void {
+    setOpen(true);
+    setEditablePathway(pathway);
+  }
```

Comment:
This should be a callback as well
```suggestion
  const openEditPathwayModal = useCallback((pathway: Pathway): void => {
    setOpen(true);
    setEditablePathway(pathway);
  }, []);
```